### DISCUSSION TITLE
Field is missing `name` attribute error when using RHF and reactstrap

### DISCUSSION THREAD

**Discussion by chris-at-lemon** [react-hook-form/react-hook-form#3787] [Q&A] [answered]
Hi all,

I've been playing around with RHF and really enjoyed it, thanks!

I am still relatively new to React so forgive me if I overlook sth silly.

I have a Reactstrap/RHF sample working just fine here:

 https://codesandbox.io/s/react-hook-forms-with-reactstrap-fgzqc
 
 I also forked and enhanced the nested input components sample from the docs to validate the inputs, also works fine:
 
 https://codesandbox.io/s/react-hook-form-nested-inputs-with-validation-sz46k

However, nested component and reactstrap do not play along nicely:

https://codesandbox.io/s/react-hook-form-nested-inputs-with-validation-and-reactstrap-cwud8

See the console throwing up  "Field is missing `name` attribute error". It does continue to work, but I'm worried that further down the line I may run into trouble. 

Any tips on this?

Thanks 👍 

Chris

**Comment by chris-at-lemon** [comment, marked as answer]
Magically a reboot fixed this problem both in my local Next.JS app and in codesandbox ... after hours of trying to figure it out. Go figure! Must have been sth. persisting in the browser. Sorry for wasting anyone's time!

**Comment by alxvallejo** [comment]
Not sure if this helps anyone but for me changing from a `Input` to an `input` tag fixes this error.
